### PR TITLE
add integration tests

### PR DESCRIPTION
**Pull request by tbischoff2**

### CODE REVIEW

File: service/src/test/resources/application-integrationtest.properties [hunk 1]
```diff
@@ -0,0 +1,36 @@
+server.error.include-message=always
+###### RDBS via JPA #####
+spring.jpa.orm=orm.xml
+#spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.open-in-view=true
+server.port=8090
```
Comment (mjacoby) [inline service/src/test/resources/application-integrationtest.properties:8]: Is this needed? I assume that random ports are used in the tests and setting a port in the config would not be needed.

**Review by mjacoby — COMMENTED** [review]
Tests themselves look good.
Is there a way to exclude the execution of integration test when building? This should be possible as integration test typically have much longer runtime than regular unit tests and are often excluded when doing a regular build.
A possible solution would be to have a separate `test` project and corresponding configuration in the parent POM similar to what we have in FA³ST Service. However, this also requires additional configuration in the POM(s) and maybe Github CI but we should be more or less able to copy that from the Service

File: service/src/test/resources/application-integrationtest.properties (hunk 1 above)
Comment (tbischoff2) [reply service/src/test/resources/application-integrationtest.properties]: You're right, random pprts are used. I removed the port.

**Comment by tbischoff2** [comment]
As the integration tests are triggered separately via the maven-failsafe-plugin, they can be skipped with "skipITs", indendepent of the project they are located in.

**Pull request merged**